src/routes/catalogue.route.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express'
+import {
+  createCatalogue,
+  deleteCatalogueById,
+  getAllCatalogues,
+  getCatalogueById,
+  updateCatalogue
+} from '../controllers/catalogue.controller.js'
+
+const catalogueRouter = Router()
+
+catalogueRouter.get('/', getAllCatalogues)
+catalogueRouter.get('/:id', getCatalogueById)
+catalogueRouter.post('/', createCatalogue)
+catalogueRouter.put('/:id', updateCatalogue)
```

Comment:
Put request is not called "updateCatalogueById" as the others requests that recieve the id as param. Also the naming of classes in this file is different from the convention used in the rest of the app